devops/git/workflow.md

```diff
@@ -8,8 +8,8 @@ Branches should be harmonious between projects and follow a set standard. Branch
 after the initial commit, with the correct protection rules in place.
 
 * `master` should be the working development branch [UNSTABLE]
-* `feature/*` branches created from master for each feature
-* `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly.
+* `feature/*` branches created from master for each feature, change or fix
```

Comment:
I still like the idea of `fix/*` branches. Perhaps we can have a vote on this and add it in the same PR if won?